packages/terra-tabs/tests/nightwatch/Tabs/tabs-spec.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
With the web driver tests added, how much of these tests are still needed?

Reply:
I will remove this file once I've got all of the tests switched over.